pyhole/core/plugin.py

```diff
@@ -178,6 +178,7 @@ def load_plugins(*args, **kwargs):
             __import__("pyhole.plugins", globals(), locals(), [plugin_name])
         except Exception, exc:
             LOG.error(exc)
+            raise
```

Comment:
This could definitely be improved -- and I'm open to all suggestions. But I do want to continue with the idea that plugin failures should have no impact on the core.